app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -271,13 +271,19 @@ def parse_host_aggregate(ha)
     def parse_tenant(tenant)
       uid = tenant.id
 
+      flavor_ids = []
+      @ems.with_provider_connection(:tenant => tenant.id) do |connection|
+        flavor_ids = connection.handled_list(:flavors).map(&:id)
```

Comment:
wonder if we do this from a flavors side with http://developer.openstack.org/api-ref-compute-v2.1.html#os-flavor-access-v2.1

so same we do for the image <-> cloud_tenants membership